pages/dashboard.js

```diff
@@ -156,18 +176,31 @@ const Dashboard = () => {
                 flexDirection={['column']}
                 alignItems='center'
                 backgroundColor='puddle'
-                padding='2.5rem 2.25rem'
+                padding='1.5rem 1.25rem'
               >
                 {donationLoading && (
                   <CircularProgress isIndeterminate color='ocean' />
                 )}
                 {!donationLoading && <Heading>$ {donation}</Heading>}
                 <Text>Monthly donation</Text>
               </ListItem>
+              <ListItem
+                className='u-box-shadow'
+                display={['none', 'flex']}
+                flexDirection={['row']}
+                alignItems='center'
+                backgroundColor='white'
+              >
+                <Button onClick={() => downloadData(JSON.stringify(userInstallData), 'flossbank_user_data.json')} 
+                        margin='1rem 1.25rem'>
+                  <Heading>Download data</Heading>
+                  <Icon marginLeft='1rem' name='external-link' size='32px' />
```

Comment:
I can change this when I start on the dash, but just a bit of a nitpicky thing: we should definitely be using the `download` icon here. `external-link` and the `download` icons have such different patterns and a user would be expecting something to link to a non-Flossbank site and often open in a new tab/window.

Reply:
Omg definitely! Tbh i couldn’t Find the download, I’ll change that now